app.rb

```diff
@@ -12,28 +12,25 @@ class Chitter < Sinatra::Base
   end
 
   get '/new' do
-  erb :'new'
+    erb :'new'
   end
 
   get '/enter' do
     redirect '/new'
   end
 
-
   post '/new' do
-  Peep.create(message: params[:message], date: Time.now.strftime("%d/%m/%Y %H:%M"))
+    Peep.create(message: params[:message],
+      date: Time.now.strftime("%d/%m/%Y %H:%M"))
     redirect '/chitter'
   end
```

Comment:
you could have potentially do the creation of a time stamp in a class for the `peep` objects?